core/src/test/java/hudson/util/XStream2Test.java

```diff
@@ -530,9 +533,42 @@ public void setListDefaultNull(List<String> listDefaultNull) {
     public void crashXstream() throws Exception {
         try {
             new XStream2().fromXML("<void/>");
-            fail("expected to throw ForbiddenClassException, but why are we still alive?");
-        } catch (ForbiddenClassException ex) {
+            fail("expected to throw ConversionException, but why are we still alive?");
+        } catch (XStreamException ex) {
             // pass
         }
     }
+
+    @Test
+    public void annotations() throws Exception {
+        assertEquals("not registered, so sorry", "<hudson.util.XStream2Test_-C1/>", Jenkins.XSTREAM2.toXML(new C1()));
```

Comment:
Just demonstrating what would happen with plugins that used annotations without a `processAnnotations` call.